resources/public/api/conf/1.0/sandbox/number-of-employees-sandbox-data.md

```diff
@@ -58,20 +58,20 @@
         <td><p>Any payload that does not meet the validation rules</p></td>
         <td>
             <p>400 (Bad Request)</p>
-            <p>{ &quot;code&quot; : &quot;INVALID_REQUEST&quot;,<br/>&quot;message&quot; : &quot;Malformed CRN submitted&quot; }</p></td>
+            <p>{ &quot;code&quot; : &quot;INVALID_REQUEST&quot;,<br/>&quot;message&quot; : &quot;Malformed payload&quot; }</p>
```

Comment:
This will likely need a code change. Beware

Reply:
yeah, we havent implemented this yet so should be fine